src/MusicStore.Standalone/project.json

```diff
@@ -5,23 +5,34 @@
   "description": "Music store application on ASP.NET 5",
   "version": "1.0.0-*",
   "buildOptions": {
-    "emitEntryPoint": true,
-    "warningsAsErrors": true,
+    "compile": {
+      "exclude": [
+        "../MusicStore/Program.cs",
+        "../MusicStore/bin",
+        "../MusicStore/obj"
+      ],
+      "include": [
+        "../../shared/**/*.cs",
+        "../MusicStore/**/*.cs"
+      ]
+    },
+    "copyToOutput": {
```

Comment:
This was the main change that was required. Other changes are just to comply to the new project.json schema.